spec/formatting.md

```diff
@@ -156,25 +202,41 @@ Its resolved value is defined by _literal resolution_.
 
 ### Literal Resolution
 
-The resolved value of a _text_ or a _literal_ is
+The _resolved value_ of a _text_ or a _literal_ contains
 the character sequence of the _text_ or _literal_
 after any character escape has been converted to the escaped character.
 
 When a _literal_ is used as an _operand_
 or on the right-hand side of an _option_,
-the formatting function MUST treat its resolved value the same
+the formatting function MUST treat its _resolved value_ the same
 whether its value was originally a _quoted literal_ or an _unquoted literal_.
 
 > For example,
 > the _option_ `foo=42` and the _option_ `foo=|42|` are treated as identical.
 
-The resolution of a _text_ or _literal_ MUST resolve to a string.
+The resolution of a _text_ or _literal_ MUST resolve with a string value.
```

Comment:
Not sure about this change? The text or literal is a string, but the resolved value might depend on the function. E.g. `{1234 :number}` had a literal which MUST be a string, but the resolved value is a number? Too pedantic?

Reply:
Note that in `{1234 :number}` we have two separate resolved values that end up being considered:
1. The resolved value of the _operand_ `1234`.
2. The resolved value of the _expression_ `{1234 :number}`.

This phrase is strictly about the former, as we're requiring that the parsing of the string `'1234'` into a numerical value can only be done by the `:number` function handler.

Reply:
Maybe clearer as:

```suggestion
The _resolved value_ of a _text_ or _literal_ is a string.
```

Reply:
Actually, we should drop this as it's duplicating the first paragraph of this section. Merge this suggestion if you agree?
```suggestion
```